src/com/hms_networks/americas/sc/historicaldata/CorruptedTimeTrackerException.java

```diff
@@ -0,0 +1,20 @@
+package com.hms_networks.americas.sc.historicaldata;
```

Comment:
I think the exceptions should be added in their own commit

Reply:
im fine with them being left in as this commit is the commit that makes use of them. They would be fine in their own commits though.

Reply:
I'm fine either way,. Adding them in their own commit I think would just provide a slightly better/cleaner git log.

Reply:
I went ahead and put them in a separate commit